include/alpaka/dev/Traits.hpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2020 Benjamin Worpitz, Bernhard Manfred Gruber
+/* Copyright 2022 Benjamin Worpitz, Bernhard Manfred Gruber, Antonio Di Pilato
```

Comment:
Did you change something in this file?

Reply:
Well, there was the native handle here but now was removed, so I'll revert back to 2020 and without my name.